### PR TITLE
{devel}[GCCcore/8.3.0] CMake v3.15.3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3151.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/ac0abcf4ab03dc81dc8b550ef1d843c8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2455.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/375973e9384773d3bc51ba81f5d70e59 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2455.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/59f47cdb6fe40e2cc5e796934ca08739 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1cb5a4521d0dc2de4151148553b098e4 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/80afc165f1968dc134b4174535b182b6 for a full test report.

File: easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+homepage = 'http://www.cmake.org'
+
+description = """
+ CMake, the cross-platform, open-source build system.  CMake is a family of
+ tools designed to build, test and package software.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
```
Comment (Micket) [inline easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb]: https?

File: easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,62 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.15.3'
+
+homepage = 'http://www.cmake.org'
```
Comment (Micket) [inline easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb]: https?

File: easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb]: fixed in fb2f08fd78

**Pull request merged**